src/services/export.service.ts

```diff
@@ -26,7 +27,34 @@ export class ExportService implements ExportServiceAbstraction {
     constructor(private folderService: FolderService, private cipherService: CipherService,
         private apiService: ApiService) { }
 
-    async getExport(format: 'csv' | 'json' = 'csv'): Promise<string> {
+    async getExport(format: 'csv' | 'json' | 'encrypted_json' = 'csv'): Promise<string> {
+        if (format === 'csv' || format === 'json') {
+            return this.getDecryptedExport(format);
+        } else {
+            return this.getEncryptedExport(format);
+        }
+    }
+
+    async getOrganizationExport(organizationId: string,
+        format: 'csv' | 'json' | 'encrypted_json' = 'csv'): Promise<string> {
+        if (format === 'csv' || format === 'json') {
```

Comment:
Maybe consider reversing the `if...else` order to manage by exception / shorter list of conditions `if (format === 'encrypted_json')`